src/TBuffer.cpp

```diff
@@ -4322,8 +4350,8 @@ void TBuffer::encodingChanged(const QString& newEncoding)
                 qDebug().nospace() << "encodingChanged(" << newEncoding << ") INFO: Installing a codec that can handle:" << mMainIncomingCodec->aliases();
             }
         } else if (mMainIncomingCodec) {
-            qDebug().nospace() << "encodingChanged(" << newEncoding << ") INFO: Uninstall a codec that can handle:" << mMainIncomingCodec->aliases() << " as the new encoding setting of: \""
-                               << mEncoding << "\" does not need a dedicated one explicitly set...";
```

Comment:
I didn't need to add quotes as that happens anyway because I did not use `qDebug().noquote()`...